lib/components/ActionSheet/types.ts

```diff
@@ -34,7 +34,6 @@ interface ActionType {
 export interface FooterProps
     extends ComponentPropsWithoutRef<"div">,
         actionSheetFooterCVA {
-    handleClose?: () => void;
```

Comment:
Reverting ActionSheet changes in this file from this [PR](https://github.com/deriv-com/quill-ui/pull/303/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519).